pages/api/home/home.tsx

```diff
@@ -207,16 +232,22 @@ const Home = (props: any) => {
         value: cleanedSelectedConversation,
       });
     } else {
-      const lastConversation = conversations[conversations.length - 1];
-      dispatch({
-        field: 'selectedConversation',
-        value: {
-          id: uuidv4(),
-          name: t('New Conversation'),
-          messages: [],
-          folderId: null,
-        },
-      });
+      // Create homepage conversation like sidebar does, but mark it as homepage conversation
+      const homepageConversation: Conversation = {
+        id: uuidv4(),
+        name: t('New Conversation'),
+        messages: [],
+        folderId: null,
+        isHomepageConversation: true, // Flag to track it's a homepage conversation
```

Comment:
`isHomepageConversation` is not part of the Conversation type
https://github.com/NVIDIA/NeMo-Agent-Toolkit-UI/blob/a0e16981069130fa6426b0496a04241e77d39e15/types/chat.ts#L20-L25

Consider adding it and either make it `nullable` or adjust the logic to handle false-y values.